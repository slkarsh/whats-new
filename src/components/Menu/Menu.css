nav {
    color: white;
    display: flex;
    flex-direction: column;
    width: 20%;
}

h1 {
    font-size: 50px;
}

ul {
    list-style: none;
}

li {
    font-size: 30px;
    margin: 15px;
}
