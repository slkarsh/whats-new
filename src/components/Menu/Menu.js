import React from 'react';
import './Menu.css'

// MENU COMPONENT CODE GOES HERE

const Menu = ({ changeCurrent }) => {
    return (
        <nav>
            <h1>What's New?</h1>
            <ul>
                <li id='local' onClick={changeCurrent}>Local News</li>
                <li id='entertainment' onClick={changeCurrent}>Entertainment</li>
                <li id='health' onClick={changeCurrent}>Health</li>
                <li id='science' onClick={changeCurrent}>Science</li>
                <li id='technology' onClick={changeCurrent}>Technology</li>
            </ul>
        </nav>
    )
}

export default Menu;