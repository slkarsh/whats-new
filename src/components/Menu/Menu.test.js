import React from 'react'
import { shallow } from 'enzyme'
import Menu from './Menu'

describe('Menu', () => {
    it('should match the snapshot when rendered correctly', () => {
        const wrapper = shallow(<Menu />)
        expect(wrapper).toMatchSnapshot();
    })
})