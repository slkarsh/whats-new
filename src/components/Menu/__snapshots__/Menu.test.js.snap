// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Menu should match the snapshot when rendered correctly 1`] = `
<nav>
  <h1>
    What's New?
  </h1>
  <ul>
    <li
      id="local"
    >
      Local News
    </li>
    <li
      id="entertainment"
    >
      Entertainment
    </li>
    <li
      id="health"
    >
      Health
    </li>
    <li
      id="science"
    >
      Science
    </li>
    <li
      id="technology"
    >
      Technology
    </li>
  </ul>
</nav>
`;
